src/models/auth.js

```diff
@@ -28,6 +43,7 @@ export default {
       };
     },
     removeUser() {
+      localStorage.removeItem('token');
```

Comment:
Interesting that `logoutUser` uses a reducer to remove the token from `localStorage` but `loginUser` doesn't use one to store the token. These should be consistent.

I really noticed this inconsistency because the reducer name is potentially confusing: it looks like we're taking some action on the _user_ itself rather than on the authorization token. It should at least be better named.

Reply:
@dbutenhof so, ideally, the logout functionality should hit a server endpoint to gracefully remove the user (also removing the token somehow).

Reply:
Well, yes; logout should call the server to remove the authorization session, too. But I was talking more about the fact that logout uses a helper to remove the `localStorage` item while login doesn't. (And the fact that `removeUser` is a misleading name since it's only removing the authorization token.)

Reply:
hey @dbutenhof, let me know if the workflow makes a little more sense now